src/app/services/transaction.service.ts

```diff
@@ -3,22 +3,38 @@ import { Subscription } from "rxjs";
 import { CreateOrder, ConfirmOrder } from "../models/paypalOrder";
 import { TransactionRepository } from "../repository/transaction.repository";
 import { WithDrawModel } from '../models/withdraw';
+import { Apollo, gql } from 'apollo-angular';
```

Comment:
```suggestion
import { Apollo } from 'apollo-angular';
```
Creo que en este archivo ya no está el gql